### PR TITLE
Add oVirt ovirt_vmpools and ovirt_vmpools_facts modules

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

- New Module Pull Request
##### COMPONENT NAME

- ovirt_vmpools
- ovirt_vmpools_facts
##### SUMMARY

This PR adds two new modules for manage VM pools in oVirt.

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina @ryansb

**Comment by gregdek** [comment]
Thanks again to @machacekondra. This module is going into community review.

We notice that you are one of the maintainers (@machacekondra) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/ovirt/ovirt_vmpools_facts.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        - "Search term which is accepted by oVirt search backend."
+        - "For example to search vmpool X: name=X"
+extends_documentation_fragment: ovirt
+'''
+
+EXAMPLES = '''
+# Examples don't contain auth parameter for simplicity,
+# look at ovirt_auth module to see how to reuse authentication:
+
+# Gather facts about all vmpools named C<rhel*>:
```
Comment (mwperina) [inline cloud/ovirt/ovirt_vmpools_facts.py]: I'd rephrase to: 'Gather facts about all vm pools which names start with C<centos>:'

**Review by mwperina — CHANGES_REQUESTED** [review]

File: cloud/ovirt/ovirt_vmpools_facts.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- ovirt_vmpools_facts:
+    pattern: name=rhel*
```
Comment (mwperina) [inline cloud/ovirt/ovirt_vmpools_facts.py]: rhel -> centos

File: cloud/ovirt/ovirt_vmpools_facts.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        - "Search term which is accepted by oVirt search backend."
+        - "For example to search vmpool X: name=X"
+extends_documentation_fragment: ovirt
+'''
+
+EXAMPLES = '''
+# Examples don't contain auth parameter for simplicity,
+# look at ovirt_auth module to see how to reuse authentication:
+
+# Gather facts about all vm pools which names start with C(centos*):
```
Comment (mwperina) [inline cloud/ovirt/ovirt_vmpools_facts.py]: 'C(centos*)' -> 'C(centos)'

**Review by mwperina — CHANGES_REQUESTED** [review]

**Review by mwperina — APPROVED** [review]
Looks good to me, thank! :+1:

**Comment by machacekondra** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @machacekondra. Since you are one of the maintainers (@machacekondra) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
Thanks again to @machacekondra. This module is going into community review.

We notice that you are one of the maintainers (@machacekondra) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by machacekondra** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @machacekondra. Since you are one of the maintainers (@machacekondra) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Review by mwperina — APPROVED** [review]
:+1:
